### PR TITLE
Release v2.0.0

### PR DESCRIPTION
**Pull request by smk4664**
# Closes: #<ISSUE NUMBER GOES HERE>

## What's Changed

Release 2.0.0

### Added

- [#67](https://github.com/nautobot/nautobot-plugin-chatops/issues/67) - Added the ability within Slack to mention the chat bot by name in a channel and in threads.
- [#194](https://github.com/nautobot/nautobot-plugin-chatops/issues/194) - Added Mattermost container for the development.
- [#194](https://github.com/nautobot/nautobot-plugin-chatops/issues/194) - Improved invoke tasks.
- [#197](https://github.com/nautobot/nautobot-plugin-chatops/issues/197) - Add a try/except block and error logger message when access_token missing from get_token() response.
- [#202](https://github.com/nautobot/nautobot-plugin-chatops/issues/202) - Added Ansible integration from nautobot-plugin-chatops-ansible.
- [#203](https://github.com/nautobot/nautobot-plugin-chatops/issues/203) - Added IP Fabric integration from nautobot-plugin-chatops-ipfabric.
- [#204](https://github.com/nautobot/nautobot-plugin-chatops/issues/204) - Added Arista CloudVision integration from nautobot-plugin-chatops-arista-cloudvision.
- [#205](https://github.com/nautobot/nautobot-plugin-chatops/issues/205) - Added Cisco Meraki integration from nautobot-plugin-chatops-meraki.
- [#206](https://github.com/nautobot/nautobot-plugin-chatops/issues/206) - Added Cisco ACI integration from nautobot-plugin-chatops-aci.
- [#207](https://github.com/nautobot/nautobot-plugin-chatops/issues/207) - Added Panorama integration from nautobot-plugin-chatops-panorama
- [#208](https://github.com/nautobot/nautobot-plugin-chatops/issues/208) - Added Grafana integration from nautobot-plugin-chatops-grafana.

### Changed

- [#67](https://github.com/nautobot/nautobot-plugin-chatops/issues/67) - Modified the clear command to not work within Slack threads.
- [#196](https://github.com/nautobot/nautobot-plugin-chatops/issues/196) - Update to microsoft IPs and correcting the one in the doc.
- [#218](https://github.com/nautobot/nautobot-plugin-chatops/issues/218) - Added `/clear` command to development Mattermost.
- [#218](https://github.com/nautobot/nautobot-plugin-chatops/issues/218) - Improved integration workers loading.
- [#218](https://github.com/nautobot/nautobot-plugin-chatops/issues/218) - Sorted App config and environment variables.
- [#218](https://github.com/nautobot/nautobot-plugin-chatops/issues/218) - Sorted pyproject configurations.

### Removed

- Drop support for Python 3.7

### CODE REVIEW

File: tasks.py [hunk 1]
```diff
@@ -34,23 +41,24 @@ def is_truthy(arg):
 
 # Use pyinvoke configuration for default values, see http://docs.pyinvoke.org/en/stable/concepts/configuration.html
 # Variables may be overwritten in invoke.yml or by the environment variables INVOKE_NAUTOBOT_CHATOPS_xxx
-
 namespace = Collection("nautobot_chatops")
 namespace.configure(
     {
         "nautobot_chatops": {
-            "nautobot_ver": "1.4.0",
+            "nautobot_ver": "1.5.24",
```
Comment (whitej6) [inline tasks.py:48]: This should match the min version of 1.5.4

File: development/Dockerfile [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
-# Version: 1.0.0
+# Version: 1.1.0
 #
 # Apps that need to add additional steps or packages can do in the section below.
 # -------------------------------------------------------------------------------------
 # !!! USE CAUTION WHEN MODIFYING LINES BELOW
 
 # Accepts a desired Nautobot version as build argument, default to 1.4.0
-ARG NAUTOBOT_VER="1.4"
+ARG NAUTOBOT_VER="1.4.0"
```
Comment (whitej6) [inline development/Dockerfile:10]: Make sure to match the minimum version

**Review by whitej6 — CHANGES_REQUESTED** [review]

File: development/Dockerfile (hunk 2 above)
Comment (smk4664) [reply development/Dockerfile]: Fixed

File: tasks.py (hunk 1 above)
Comment (smk4664) [reply tasks.py]: Fixed

**Review by whitej6 — APPROVED** [review]
Thank you for the updates!!

**Pull request merged**